api/src/main/java/org/openmrs/module/aijar/program/ExitMCHProgramTask.java

```diff
@@ -0,0 +1,58 @@
+package org.openmrs.module.aijar.program;
+
+import java.util.Date;
+import java.util.List;
+
+import org.joda.time.DateTime;
+import org.joda.time.Days;
+import org.openmrs.Encounter;
+import org.openmrs.Patient;
+import org.openmrs.PatientProgram;
+import org.openmrs.Program;
+import org.openmrs.api.ProgramWorkflowService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.aijar.metadata.core.Programs;
+import org.openmrs.scheduler.tasks.AbstractTask;
+
+/**
+ * Exits patients, who have lost follow up, from the MCH program
+ */
+public class ExitMCHProgramTask extends AbstractTask {
```

Comment:
If this is exiting for only lost to followup - the class name should be ExitMCHProgramASLostToFollowUpTask